demos/object_detection_demo/python/models.lst

```diff
@@ -48,3 +48,4 @@ yolo-v2-tiny-tf
 yolo-v2-tiny-vehicle-detection-0001
 yolo-v3-tf
 yolo-v3-tiny-tf
+# For --architecture_type=yolov4
```

Comment:
Since you've updated tests, you should extend this list.

Reply:
Done